.gitignore

```diff
@@ -4,3 +4,4 @@ dist/*
 .vscode/*
 gitsign
 gitsign-credential-cache
+!/cmd/gitsign-credential-cache
```

Comment:
```suggestion
/gitsign-credential-cache
```

This will just exclude the built binary, and not any path matching `*gitsign-credential-cache*`

Reply:
This will only ignore it if it is in the root directory, which won't exclude binaries built if running `go build .` from the subdirectory itself. 😢

Reply:
booooooo